packages/shared/src/lib/core/profile/migrations/prod/prod-profile-migration-map.ts

```diff
@@ -29,5 +30,6 @@ export const PROD_PROFILE_MIGRATION_MAP: ProfileMigrationMap = {
     10: prodProfileMigration10To11,
     // ^^^ release 1.0.4 ^^^
     11: prodProfileMigration11To12,
+    12: prodProfileMigration12To13,
     // ^^^ release 1.0.8 ^^^
```

Comment:
this migration is part of the next release
```suggestion
    // ^^^ release 1.0.8 ^^^
    12: prodProfileMigration12To13,
```